src/display/api.js

```diff
@@ -2705,6 +2705,7 @@ class WorkerTransport {
       .then(results => {
         return {
           info: results[0],
+          rawMetadata: results[1],
```

Comment:
Can you please remove this property, and instead do something like this:
 1. Add `this._data = data;` after line https://github.com/mozilla/pdf.js/blob/c0a6d6cd212436a2ed4e796815fc11d04d94c638/src/display/metadata.js#L34
 2. Add a new method, e.g.
    ```javascript
    getRaw() {
      return this._data;
    }
    ```
    after line https://github.com/mozilla/pdf.js/blob/c0a6d6cd212436a2ed4e796815fc11d04d94c638/src/display/metadata.js#L126